File: reverseArray/reverseArray.js
function reverseArray (arr) {
    return arr.reverse();
}

let arr1 = [1, 2, 3, 4, 5];
console.log(reverseArray(arr1));  // Output: [5, 4, 3, 2, 1]

let arr2 = ["apple", "banana", "orange", "grape"];
console.log(reverseArray(arr2));  // Output: ['grape', 'orange', 'banana', 'apple']

let arr3 = [];
console.log(reverseArray(arr3));  // Output: []

let arr4 = [42];
console.log(reverseArray(arr4));  // Output: [42]

let arr5 = [1, "two", 3.0, [4, 5], { key: "value" }];
console.log(reverseArray(arr5));  // Output: [ { key: 'value' }, [ 4, 5 ], 3, 'two', 1 ]

File: reverseArray/reverseArray.py
def reverseArray(a):
    return a[::-1]


# Test case 1: Reversing an array of integers
arr_int = [1, 2, 3, 4, 5]
print(reverseArray(arr_int))  # Output: [5, 4, 3, 2, 1]

# Test case 2: Reversing an array of strings
arr_str = ["apple", "banana", "orange", "grape"]
print(reverseArray(arr_str))  # Output: ['grape', 'orange', 'banana', 'apple']

# Test case 3: Reversing an empty array
arr_empty = []
print(reverseArray(arr_empty))  # Output: []

# Test case 4: Reversing an array with one element
arr_single = [42]
print(reverseArray(arr_single))  # Output: [42]

# Test case 5: Reversing an array with mixed data types
arr_mixed = [1, "two", 3.0, [4, 5], {"key": "value"}]
print(reverseArray(arr_mixed))  # Output: [{'key': 'value'}, [4, 5], 3.0, 'two', 1]
